perf_test/ode/CMakeLists.txt

```diff
@@ -2,22 +2,7 @@ KOKKOSKERNELS_INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
 KOKKOSKERNELS_INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR})
 
 if(KOKKOSKERNELS_ENABLE_BENCHMARK)
-  SET(BENCHMARK_NAME ${PACKAGE_NAME}_ode_runge_kutta)
-
-  ADD_EXECUTABLE(
-      ${BENCHMARK_NAME}
-      KokkosODE_RK.cpp
-  )
-  TARGET_LINK_LIBRARIES(
-      ${BENCHMARK_NAME}
-      PRIVATE benchmark::benchmark Kokkos::kokkoskernels
-  )
-  TARGET_INCLUDE_DIRECTORIES(
-      ${BENCHMARK_NAME}
-      SYSTEM PRIVATE ${benchmark_SOURCE_DIR}/include
-  )
-  ADD_TEST(
-      NAME ${BENCHMARK_NAME}
-      COMMAND ${BENCHMARK_NAME}
+  KOKKOSKERNELS_ADD_BENCHMARK(
+    ode_runge_kutta SOURCES KokkosODE_RK.cpp
```

Comment:
Drive-by change to use the common helper (it no longer add `BenchmarkMain.cpp`).